### ISSUE TITLE
Dev: Update react-scripts dependency to remove high severity vulnerabilities

### ISSUE THREAD

**Issue by mtreacy002** [anitab-org/bridge-in-tech-web#194]
### Is your feature request related to a problem? Please describe.
The current react-scripts dependency (v3.4.3) is causing 2 high severity warnings

### Describe the solution you'd like
Update react-scripts dependency from 3.4.3 to 4.0.3. Note: this install involves potential breaking changes.
While on it, fix other vulnerability warnings as well.

### Describe alternatives you've considered
Ignore warnings if they don't causing major issue

### Additional context
These warnings were caught on the initial dependencies installation of a newly cloned project.
Here's the [gist](https://gist.github.com/mtreacy002/592b670535b1d367a867615365f1aa33) of npm audit reports.

**Comment by Amulya-coder** [comment]
@mtreacy002 I have added the `Open source hack `  label for the OSH aspirants

**Comment by Anmollenka** [comment]
I would like to work on this issue.

**Comment by Amulya-coder** [comment]
Assigning @Anmollenka.

**Comment by Anmollenka** [comment]
@mtreacy002  just wanted to confirm if medium severity vulnerabilities should be removed too?

**Comment by Anmollenka** [comment]
> @mtreacy002 just wanted to confirm if medium severity vulnerabilities should be removed too?

@Amulya-coder  Can you just confirm so that I can create a pull request.

**Comment by Amulya-coder** [comment]
@Anmollenka, Yes you can go ahead and create a pull request no need to worry about medium severity vulnerabilities.

**Comment by vj-codes** [comment]
@Anmollenka any updates here?

**Comment by Anmollenka** [comment]
Will create a pr within an hour

**Comment by Anmollenka** [comment]
@vj-codes After manually changing the versions 2 high vulnerabilities are still there.

**Comment by mtreacy002** [comment]
@Anmollenka , can you please show us steps you've done along with the log on a gist of npm audit report? This will help us better understand the issue you are facing. Thanks 😉

**Comment by Anmollenka** [comment]
sure @mtreacy002

**Comment by Anmollenka** [comment]
@mtreacy002  I updated the react version to 4.0.3 and ran npm audit.Here is the  [gist](https://gist.github.com/Anmollenka/71bb711062be542deb52655d2e93ea0d) of the npm audit report.

**Comment by mtreacy002** [comment]
@Anmollenka , when you said you've manually changed the versions, which versions you're talking about? for example, I can't see how you've updated react version to 4.0.3 while the version stated inside package.json is 16.3.1.
<img width="581" alt="Screen Shot 2021-07-07 at 8 23 00 pm" src="https://user-images.githubusercontent.com/29667122/124743693-51ac4c00-df61-11eb-9469-51eed9e8aea9.png">

can you please submit the PR with whatever you currently have so that we could see what you have done and how we can improve this? thanks

**Comment by Anmollenka** [comment]
> @Anmollenka , when you said you've manually changed the versions, which versions you're talking about? for example, I can't see how you've updated react version to 4.0.3 while the version stated inside package.json is 16.3.1.
> <img alt="Screen Shot 2021-07-07 at 8 23 00 pm" width="581" src="https://user-images.githubusercontent.com/29667122/124743693-51ac4c00-df61-11eb-9469-51eed9e8aea9.png">
> 
> can you please submit the PR with whatever you currently have so that we could see what you have done and how we can improve this? thanks

Yes @mtreacy002 as I have not submitted my pull request you will not be able to see my changes.Sorry for the inconvenience caused by not explaining my approach properly.
